app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -433,6 +446,19 @@ public void onPause() {
     @Override
     public void onStop() {
         super.onStop();
+
+        if (quietMode) {
+            sharedPreference.save(getActivity(), PREFS_QUIET, true);
+        } else {
+            sharedPreference.save(getActivity(), PREFS_QUIET, false);
+        }
+
+        if (shakeOn) {
+            sharedPreference.save(getActivity(), PREFS_SHAKE, false);
+        } else {
+            sharedPreference.save(getActivity(), PREFS_SHAKE, true);
+        }
```

Comment:
I don't think you need this block of code because quiteMode and shakeOn are already booleans so you should just be able to say:
sharedPreference.save(getActivity(), PREFS_SHAKE, onShake);
and
sharedPreference.save(getActivity(), PREFS_QUIET, quiteMode);